<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PDWIVEDI" name="com.ms.sapphire.etl.AERS_REACTIONS" tests="3" failures="0" timestamp="2 Jul 2014 15:35:03 GMT" time="0.247" errors="1">
  <testcase name="reaction_VBM" time="0.192" classname="com.ms.sapphire.etl.AERS_REACTIONS">
    <error type="java.lang.AssertionError" message="pt_name is null in aers_reactions for [41]records">
      <![CDATA[java.lang.AssertionError: pt_name is null in aers_reactions for [41]records
	at org.testng.Assert.fail(Assert.java:94)
	at com.ms.sapphire.etl.AERS_REACTIONS.reaction_VBM(AERS_REACTIONS.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:368)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:662)
]]>
    </error>
  </testcase> <!-- reaction_VBM -->
  <testcase name="reaction_pt_code_notnull" time="0.030" classname="com.ms.sapphire.etl.AERS_REACTIONS"/>
  <testcase name="reaction_pt_name_notnull" time="0.025" classname="com.ms.sapphire.etl.AERS_REACTIONS"/>
</testsuite> <!-- com.ms.sapphire.etl.AERS_REACTIONS -->
